tools/syz-execprog/execprog.go

```diff
@@ -32,9 +33,11 @@ var (
 	flagRepeat    = flag.Int("repeat", 1, "repeat execution that many times (0 for infinite loop)")
 	flagProcs     = flag.Int("procs", 1, "number of parallel processes to execute programs")
 	flagOutput    = flag.Bool("output", false, "write programs and results to stdout")
+	flagHints     = flag.Bool("hints", false, "do a hints-generation run")
 	flagFaultCall = flag.Int("fault_call", -1, "inject fault into this call (0-based)")
 	flagFaultNth  = flag.Int("fault_nth", 0, "inject fault on n-th operation (0-based)")
-	flagHints     = flag.Bool("hints", false, "do a hints-generation run")
+	flagEnable    = flag.String("enable", "none", "enable only listed additional features")
```

Comment:
The list of features needs to appear in the help message. It's very handy to run `syz-execprog -help` or `syz-prog2c -help` to get a list of what can be enabled/disabled. "enable only listed additional features" is not actionable.

Reply:
Is there an easy way to do this with the `flag` package? Like a dynamically generated custom message for one of the flags.

Reply:
Fixed.